tests/test_auth.py

```diff
@@ -40,7 +40,7 @@ def test_returns_resource(self, _is_authenticated_request):
 class TestIsAuthenticatedRequest(object):
 
     def test_returns_false_by_default(self):
-        assert is_authenticated_request() == False
+        assert is_authenticated_request() is False
```

Comment:
I know but it doesn't change the type. 
- not 'a' -> False
- not 5 - > False
- ...